src/libraries/System.Net.Quic/tests/FunctionalTests/QuicTestBase.cs

```diff
@@ -44,14 +44,20 @@ public abstract class QuicTestBase : IDisposable
 
         static unsafe QuicTestBase()
         {
-            Console.WriteLine($"MsQuic {(IsSupported ? "supported" : "not supported")} and using '{MsQuicApi.MsQuicLibraryVersion}'.");
+            // If any of the reflection bellow breaks due to changes in "System.Net.Quic.MsQuicApi", also check and fix HttpStress project as it uses the same hack.
```

Comment:
I'm wondering if this should go somewhere to `Common/test/....` so we can avoid duplication.
It may be worth of adding explanation as this is unusual craft.

Reply:
Sounds good, I'll look into it.

Reply:
We should keep in mind that whatever utility helper we add, its! file has to be also copied to the container, so avoiding duplication has a higher cost in complexity here than normally. At the moment I'm undecided if it's worth it, would prefer to do the docker copying work mentioned in https://github.com/dotnet/runtime/pull/84793#discussion_r1172373409 first, planning to jump to it tomorrow.

Reply:
Ok, I'm holding back. Let me know when/if I should look into it.